packages/helm/symphony/files/oss/delete-objects.sh

```diff
@@ -1,29 +1,117 @@
 #!/bin/env bash
 set -e
-TIMEOUT="30m"
+TIMEOUTINSTANCE="15m"
+TIMEOUT="1m"
+TIMEOUTFINALIZER=60
```

Comment:
@FireDefend , @RemindD , please review whether the TIMEOUT is sufficient for target, instance, campaign, activation deletion.